_includes/head.html

```diff
@@ -13,4 +13,8 @@
   {% if jekyll.environment == 'production' and site.google_analytics %}
   {% include google-analytics.html %}
   {% endif %}
+
+  {% if site.gems contains 'jekyll-seo-tag' %}
+  {% include seo.html %}
```

Comment:
> Also, this solution wont work in the edge-situation where the user has the plugin loaded only via the `:jekyll_plugins` group in `Gemfile`

I don't know that I'd call that an edge case, as it is one of three official ways to install a plugin, as [documented on jekyllrb.com](https://jekyllrb.com/docs/plugins/#installing-a-plugin)